src/components/Admin/verification/validateDriver.js

```diff
@@ -0,0 +1,5 @@
+import camlecase from 'camelcase';
+
+export default function validateDriver(driver) {
+  return Object.fromEntries(Object.entries(driver).map((item) => [camlecase(item[0]), item[1]]));
```

Comment:
`.map((item) => [camlecase(item[0]), item[1]]));` to not have array indexes here, destruct in map params section, `.map([name1, name2])`, and also name those variables with field names